xbmc/TextureCacheJob.cpp

```diff
@@ -32,7 +32,7 @@
 #include "FileItem.h"
 #include "music/MusicThumbLoader.h"
 #include "music/tags/MusicInfoTag.h"
-#if defined(HAS_OMXPLAYER)
+#if defined(TARGET_RASPBERRY_PI)
```

Comment:
+1 As I already said before, this doesn't make any sense. Either we do this consistently or we don't.